buildenv/1.3.x/win32-static/libogg.build

```diff
@@ -1,28 +1,27 @@
 #!/usr/bin/env mumble-build
-# Copyright 2013-2015 The 'mumble-releng' Authors. All rights reserved.
+# Copyright 2013-2018 The 'mumble-releng' Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that
 # can be found in the LICENSE file in the source tree or at
 # <http://mumble.info/mumble-releng/LICENSE>.
 
 urls=(
-	"http://releng-distfiles.mumble.info/releng-distfiles/libogg-1.3.2.tar.gz"
-	"http://downloads.xiph.org/releases/ogg/libogg-1.3.2.tar.gz"
+	"https://s3.amazonaws.com/mumble-releng-distfiles/libogg-1.3.3.tar.gz"
+	"https://downloads.xiph.org/releases/ogg/libogg-1.3.3.tar.gz"
 )
 
 digests=(
-	"sha1:df7f3977bbeda67306bc2a427257dd7375319d7d"
-	"sha256:e19ee34711d7af328cb26287f4137e70630e7261b17cbe3cd41011d73a654692"
+	"sha1:28ba40fd2e2d41988f658a0016fa7b534e509bc0"
+	"sha256:c2e8a485110b97550f453226ec644ebac6cb29d1caef2902c007edab4308d985"
+	"blake2:7b229408df807da06a697e55a4c58599bae30ff57b6245f0e723a9896f29c48a98e6874d1147452cf5b4e992f21b3bd441b758e8743a250bc7fe552700a4ddc1"
 )
 
 function extract {
-	tar -zxf libogg-1.3.2.tar.gz
-	cd libogg-1.3.2
+	tar -zxf libogg-1.3.3.tar.gz
+	cd libogg-1.3.3
 }
 
 function prepare {
-	patch -p1 < ${MUMBLE_BUILDENV_ROOT}/patches/libogg-static-vs2010-Zi.patch
```

Comment:
Is it safe to drop this without losing debug symbols?

Reply:
I don't think so, I have to update the patch.